src/api/endpoints.js

```diff
@@ -341,6 +341,8 @@ export class api {
 
     if (season !== undefined && team === undefined) {
       addedParams = '/' + season
+    } else if (year !== undefined) {
```

Comment:
I haven’t tested but this doesn’t feel (look right) I think I need to re-write how this works.  I

hindsight says I should have used typescript!